dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -765,17 +764,17 @@ DDS::ReturnCode_t DomainParticipantImpl::delete_contentfilteredtopic(
     if (DCPS_debug_level > 3) {
       ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) ERROR: ")
         ACE_TEXT("DomainParticipantImpl::delete_contentfilteredtopic, ")
-        ACE_TEXT("can't delete a content-filtered topic \"%C\" ")
+        ACE_TEXT("can't delete a contentfiltered topic \"%C\" ")
```

Comment:
Every DDS entity that uses the topic now calls add_entity_ref, so it could be also that a datawriter is still using the topic, not purely a reader, so we can't say anymore that a reader uses it